guide/additional-info/reactions.md

```diff
@@ -242,10 +242,24 @@ const message = await channel.fetchMessage(data.message_id);
 
 The if statement in the middle plays an important role; it prevents us from re-emitting the event for both uncached *and* cached messages. Without this, your `messageReactionAdd` event would execute twice for a single reaction if the message was already cached.
 
-All that's left is to fetch the actual reaction from the message and emit the event. Since custom emoji reactions are keyed by their ID and unicode emoji are keyed by their name, you''ll have to do something like this:
+A custom emoji contains both a name and an ID, while a unicode emoji contains just a name. Since custom emoji reactions are keyed by their name:ID and unicode emoji reactions are keyed by their name, you'll have to do something like this to set the right emoji for this event:
```

Comment:
"custom emoji reactions are keyed by their name:ID ..." is a bit strange to read. I'd change this to something along the lines of "custom emoji reactions are keyed in a `name:ID` format".